test/e2e/obsctl_test.go

```diff
@@ -31,7 +33,22 @@ const (
 // Hydra is spun up externally via start_hydra.sh, as accessing it via docker network is difficult for obsctl.
 // Follows similar pattern as https://observatorium.io/docs/usage/getting-started.md/.
 // Also registers tenants in hydra.
-func preTest(t *testing.T) *e2e.DockerEnvironment {
+
+func hydra() *exec.Cmd {
+
+	cmd := exec.Command("/bin/sh", "start_hydra.sh")
```

Comment:
I think you need to provide the correct path for this here so that it can execute the correct script. I would suggest using `os.Getwd()` and attaching the start_hydra.sh script to that path! 🙂